index.html

```diff
@@ -320,12 +320,20 @@ <h2 class="heading-contact">
 
         </div>
     </section>
+    <script src="./script.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
         integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
         crossorigin="anonymous"></script>
     <script src="https://kit.fontawesome.com/f6777e9957.js" crossorigin="anonymous"></script>
-
+    <script>(function (w, d, s, l, i) {
+            w[l] = w[l] || []; w[l].push({
+                'gtm.start':
+                    new Date().getTime(), event: 'gtm.js'
+            }); var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+        })(window, document, 'script', 'dataLayer', 'GTM-PCX82M9');</script>
```

Comment:
Great work! Great animation. At the moment the app is not performing as expected. When I click on the `hamburger button` the mobile menu appears but when I click on any of the mobile menu options, the `hamburger button` disappears. Kindly fix this issue.

Before click | After click
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/156013448-05e763f2-7787-4e1e-937e-089f496c9c2e.png) | ![image](https://user-images.githubusercontent.com/10332499/156013538-9476fd20-e1b4-4951-ad87-b6ab2cdf0a89.png)